src/test/fhir/TestPatient.js

```diff
@@ -12,27 +12,46 @@ import { config } from './test.config.js';
 export function TestPatient() {
   let params = useParams();
   const [patientData, setPatientData] = useState([]);
-  const dashboardInput = useCds(patientData);
-  
+
+  const [toggleStatus, setToggleStatus] = useState({
+    isImmunosuppressed: false,
+    isPregnant: false,
+    isPregnantConcerned: false,
+    isSymptomatic: false,
+    isToggleChanged: false
+  });
+  const {output: dashboardInput, isLoadingCdsData } = useCds(patientData, toggleStatus);
+  const isLoading = isLoadingCdsData;
   // Extract the data for the requested test patient
   if (params.testName in testData) {
     if (patientData.length === 0) {
-      const newData = testData[params.testName].entry.reduce((acc,cv) => {
-        return [...acc, cv.resource];
-      },[])
+      // Assumes all test patient data is in a Bundle
+      const newData = testData[params.testName].entry.map((c) => {
+        if (!c.resource) return;
```

Comment:
Line 30 is throwing the following eslint warning:

> Array.prototype.map() expects a return value from arrow function.eslint [array-callback-return](https://eslint.org/docs/rules/array-callback-return)

Should we avoid unexpected behavior here by making sure we are not adding undefined values to the array of `patientData`?

Reply:
will provide empty return value if it isn't a resource

Reply:
I like adding the empty object to the patientData array in the case that c.resource doesn't exist, as opposed to undefined. Fix looks good.